src/main/java/mate/controller/DriverToCarPageController.java

```diff
@@ -0,0 +1,17 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class DriverToCarPageController extends HttpServlet {
```

Comment:
Use informative names for your controllers. Always start controller’s name with a verb: GetAllManufacturersController, AddDriverToCarController etc.

Reply:
https://mate-academy.github.io/jv-program-common-mistakes/java-web/web-practice/java-web-practice

Reply:
Done